nvtabular/loader/tensorflow.py

```diff
@@ -270,6 +275,13 @@ def _get_device_ctx(self, dev):
     def _split_fn(self, tensor, idx, axis=0):
         return tf.split(tensor, idx, axis=axis)
 
+    def _tensor_split(self, tensor, idx, axis=0):
```

Comment:
What API is this function matching? 🤔

Reply:
good catch... I needed it when the abstraction required splitting in the backend module... but that is no longer the case.

Reply:
actually this is used it is the api used in the handle_tensor function in the backend code. I was confusing it for something I had in the sparse_tensors logic.